src/App.vue

```diff
@@ -496,6 +497,9 @@ export default {
       // now that we have business info, load state filing
       await this.loadStateFiling()
 
+      // now that we have business info, load the registration filing
+      await this.loadRegistrationFiling()
```

Comment:
Can't you just use the current parties that were fetched on line 487 and stored on line 495?

Reply:
I believe those only return the business parties so I wouldn't have info on who the filing party was.

Reply:
OK. Resolving.